concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/BakerPoolStatus.java

```diff
@@ -0,0 +1,76 @@
+package com.concordium.sdk.responses.poolstatus;
+
+import com.concordium.sdk.responses.BakerId;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.AccountAddress;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Optional;
+
+@Data
+@Getter
+@ToString
+public class BakerPoolStatus extends PoolStatus {
```

Comment:
Make fields final please. 
I don't see the need for `@Getter` `@ToString` when `@Data` is used.

Reply:
done

Reply:
done